build/scripts/dist.xml

```diff
@@ -197,7 +197,14 @@
         <touch file="${dist.dir}/log/.DO_NOT_DELETE"/>
 
         <filter token="database" value="native"/>
-
+        <mkdir dir="${dist.dir}/autodeploy"/>
+        <mkdir dir="${dist.dir}/lib/test"/>
+        <copy todir="${dist.dir}/autodeploy">
```

Comment:
the autodeploy directory can be best copied with the fileset in the copy task, see line 201